Diff/diff.py

```diff
@@ -38,10 +38,15 @@ def is_visible(self, files):
 class DiffChangesCommand(sublime_plugin.TextCommand):
     def run(self, edit):
 
-        fname = self.view.file_name();
+        fname = self.view.file_name()
+
+        if not fname or not os.path.exists(fname):
+            sublime.status_message("The file does not exists")
```

Comment:
Minor nitpick, the `s` at the end of `exist` is unnecessary :)
Maybe it would be better to say "unable to diff changes; the file does not exist", to make it clearer where the message comes from?